src/containers/main-page/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Box } from '@mui/material';
+import { FormWrapperStyled, MainPageContainerStyled, PreviewWrapperStyled } from './styled';
+import CVForm from './cv-form';
+import CvFormStepperProvider from './cv-form/state/CVFormStepperProvider';
+
+const MainPage = function (): JSX.Element {
+  return (
+    <MainPageContainerStyled container>
+      <FormWrapperStyled
+        item
+        xs={12}
+        sm={12}
+        md={12}
+        lg={12}
+        xl={6}
```

Comment:
Just a question. Is it possible to place these dimentions into the styled component? I don't know.

Reply:
It's not possible, the only one thing I can perform is getProps-like pattern: {...getProps()}
const getGridHalfProps = () => ({ xs: 12, sm: 12, md: 12, xl: 6 })